apps/daimo-mobile/src/view/shared/SearchHeader.tsx

```diff
@@ -89,7 +93,14 @@ export function SearchHeader({
   return (
     <View style={styles.header}>
       <Animated.View key="back" style={backButton}>
-        <TouchableOpacity onPress={() => Keyboard.dismiss()} hitSlop={16}>
+        <TouchableOpacity
+          onPress={() => {
+            setPrefix(undefined);
+            Keyboard.dismiss();
+            ref.current?.closeInput();
```

Comment:
could we avoid the whole `forwardRef` with something like

```ts
useEffect(() => {
  if (prefix === undefined) animateCloseInput();
}, [prefix]);
```

..insider AnimatedSearchInput?

Reply:
I though the problem here was that it is parsed as `value={prefix || ""}` because it needs to always use a string but I think we can change that and you are right that we can remove forwardRef because of that.

Thanks for pointing that out 🫡